code/modules/mob/living/carbon/carbon.dm

```diff
@@ -882,6 +882,85 @@
 	cure_all_traumas(TRAUMA_RESILIENCE_MAGIC)
 	..()
 
+#define CPR_PANIC_SPEED (0.8 SECONDS)
+
+/// Performs CPR on the target after a delay.
+/mob/living/carbon/proc/do_cpr(mob/living/carbon/target, tool_assisted = FALSE)
```

Comment:
Any reason this was moved up to carbon ?

Reply:
Non-humans can do CPR with the tool, like a golem.